README.md

```diff
@@ -10,11 +10,57 @@ Package that provides various bindings to enable interoperability for external l
 
 ## Features
 
-Coming Soon!
+### DijitWrapper
+
+`DijitWrapper` is a mixin class that can convert a Dojo 1 based Dijit and allow it to integrate to the Dojo 2 widgeting system.
+
+The wrapper takes a Dijit constructor function as its input and returns a Dojo 2 widget.  For example, to take the `dijit/Calendar`
+and place it a Dojo 2 `App` widget would look something like this:
+
+```ts
+import * as CalendarDijit from 'dijit/Calendar';
+import DijitWrapper from '@dojo/interop/dijit/DijitWrapper';
+import { v, w } from '@dojo/widget-core/d';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+
+const Calendar = DijitWrapper(CalendarDijit);
+
+class App extends WidgetBase {
+  private _onCalendarChange = (date: Date) => {
+    console.log('Date selected:', date);
+  }
+
+  render() {
+    return v('div', { key: 'root' }, [
+      w(Calendar, {
+        key: 'calendar1',
+        id: 'calendar1',
+        onChange: this._onCalendarChange
+      })
+    ]);
+  }
+}
+
+export default App;
+```
+
+This would result, when displayed through a projector, the Dijit Calendar which every time the date changed, it would log the new date
```

Comment:
This would result, when displayed through a projector, the Dijit Calendar which every time the date changed, it would log the new date  to the console. -> The result, when displayed through a projector, is a Dijit Calendar which will log the new date to the console every time the date is changed.